larva/src/main/java/org/frankframework/extensions/test/IbisTester.java

```diff
@@ -37,7 +37,7 @@
 import org.frankframework.core.Adapter;
 import org.frankframework.lifecycle.FrankApplicationInitializer;
 import org.frankframework.stream.Message;
-import org.frankframework.testtool.TestTool;
```

Comment:
Als we de class hernoemen, waarom dan niet het hele package?
Er is namelijk ook verwarring tussen de Larva packages en de Ladybug packages die allebei iets met `testtool` of `ibistesttool` hebben.

Reply:
Done.